### PR TITLE
[ng] src reorg of pirate badge (ng2) codelab

### PR DESCRIPTION
**Pull request by chalin**
The corresponding code changes are covered by https://github.com/dart-lang/one-hour-codelab/pull/42

Contributes to #713

### CODE REVIEW

File: src/_codelabs/ng2/1-skeleton.md [hunk 1]
```diff
@@ -438,6 +441,9 @@ transformers:
 - angular2:
     entry_points: web/main.dart
 - dart_to_js_script_rewriter
+web:
```
Comment (kwalrath) [inline src/_codelabs/ng2/1-skeleton.md]: I think we need to keep the pubspec as close to the template as possible. For now, that means commenting out the `web` stuff.

File: src/_codelabs/ng2/1-skeleton.md [hunk 2]
```diff
@@ -470,6 +476,12 @@ transformers:
 * Pub uses the `dart_to_js_script_rewriter` transformer when building
   your app for deployment.
 
+* The `web: compiler: ...` entry tells pub to compile using the
```
Comment (kwalrath) [inline src/_codelabs/ng2/1-skeleton.md]: Let's comment this out for now.

File: src/_codelabs/ng2/1-skeleton.md (hunk 1 above)
Comment (chalin) [inline src/_codelabs/ng2/1-skeleton.md]: But we're not using the templates anymore. The code for this codelab is kickstarted from quickstart, and the quickstart's pubspec has the `web: compiler: ...` entry.

File: src/_codelabs/ng2/1-skeleton.md [hunk 3]
```diff
@@ -470,6 +476,12 @@ transformers:
...
+  `dartdevc` when either building or serving an app in `debug` mode. See
+  [Building with dartdevc][ddc] for details.
+  
+  [ddc]: https://webdev.dartlang.org/tools/dartdevc#building-with-dartdevc
```
Comment (kwalrath) [inline src/_codelabs/ng2/1-skeleton.md]: Let's make this a little shorter:

* The `web: compiler: ...` entry tells pub to use the Dart development compiler
   ([dartdevc][ddc]) when building or serving the app in `debug` mode.

Comment (chalin) [inline src/_codelabs/ng2/1-skeleton.md]: Fixed

**Pull request merged**